bindings/rust/s2n-tls/src/connection.rs

```diff
@@ -914,6 +914,15 @@ impl Connection {
         }?;
         Ok(self)
     }
+
+    /// Allows the quic library to check if session tickets are expected
+    pub fn client_resumption_enabled(&self) -> bool {
```

Comment:
I'm a fan of naming bool methods yes/no questions. So maybe "is_client_resumption_enabled"? Or "is_session_resumption_enabled" to match the more standard feature name?